src/service/randomNumberInRange.js

```diff
@@ -0,0 +1,44 @@
+// min should be a minimal number, max should be a maximal number, forbiddenNumbersArray should be an array containing all the numbers that should not be generated
+
+export const randomNumberInRange = (min, max, forbiddenNumbersArray = []) => {
+    
+    if (typeof min === 'number' && typeof max === 'number' && Array.isArray(forbiddenNumbersArray) && min <= max) {
+
+        // check if not every possible number is forbidden
+        if (max - min + 1 === forbiddenNumbersArray.length) {
+
+            const allPossibleOutComes = [];
+
+            for (let i = min; i <= max; i++) {
+                allPossibleOutComes.push(i);
+            }
+
+            const allPossibleOutComesSorted = allPossibleOutComes.sort((a, b) => {return a-b});
+            const forbiddenNumbersArraySorted = forbiddenNumbersArray.sort((a, b) => {return a-b});
```

Comment:
[Readability and function reuse]
I see that you want to sort two arrays, both the same way. It would be clearer for reader that you use some function, for example:
Instead of:
```
(a, b) => {return a-b}
```
It would be better to define sorting function above and name it accordingly:
```
const numbersAscending = (a,b) => a-b;
```
even parenthesis `{ }` and `return` keyword can be omitted.
By doing it this way you can write:
```
array.sort(numbersAscending);
```
and now you are achieving that somebody knows, that `a-b` means numbers in ascending order :)